webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7313,7 +7361,25 @@ var CURRENT_GROUP_DATA={};
                 };
                 return conversationDetail;
             };
-
+            _this.checkBotDetail = function () {
+                window.Applozic.ALApiService.ajax({
+                    url: MCK_BOT_API + "/application/" + MCK_APP_ID + "/bot/" + CURRENT_GROUP_DATA.conversationAssignee,
+                    type: 'get',
+                    global: false,
+                    success: function (data) {
+                              var  botData = data.data[0];
+                                if (botData.aiPlatform == 'dialogflow' && (!botData.autoHumanHandoff)) {
+                                    CURRENT_GROUP_DATA.CHAR_CHECK = true;
```

Comment:
If else can be replaced with one line statement

Reply:
check new commit